File: ExpenseSplitter.Frontend/ExpenseSplitter.Web/src/app/models/expense/expense-extract-model.ts
import { ExpenseTypeEnum } from 'src/app/data/expense-type';

export class ExpenseExtractModel {
    id: number;
    name: string;
    type: ExpenseTypeEnum;
    paidAt: Date;
    payerName: string;
    isPaidByMe: boolean;
    value: number;
}

File: ExpenseSplitter.Backend/ExpenseSplitter.Api/Models/Expenses/ExpenseExtractModel.cs

using System;

namespace ExpenseSplitter.Api.Models.Expenses
{
    public class ExpenseExtractModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public ExpenseType Type { get; set; }
        public DateTime PaidAt { get; set; }
        public string PayerName { get; set; }
        public bool IsPaidByMe { get; set; }
        public decimal Value { get; set; }
    }
}
